rust/frontend/src/types/errors.rs

```diff
@@ -26,6 +26,8 @@ pub enum ValidationError {
     UpdateCollection(#[from] UpdateCollectionError),
     #[error("Error parsing collection configuration: {0}")]
     ParseCollectionConfiguration(#[from] CollectionConfigurationToInternalConfigurationError),
+    #[error("Please only set space configuration for vector index type {0}")]
```

Comment:
make the error more descriptive, something along the lines "it is not recommneded to set index params other than space but instead rely on values the system automatically chooses for you for best performance and accuracy"